sensor/extrinsic_reflector_based_calibrator/extrinsic_reflector_based_calibrator/calibrator_ui.py

```diff
@@ -106,6 +125,15 @@ def add_lidar_radar_pair_status_callback(self, status):
         self.add_lidar_radar_pair_status = status
         self.check_status()
 
+    def delete_lidar_radar_pair_result_callback(self, result):
+        self.pending_service = False
+        time.sleep(1)
```

Comment:
What is the meaning of this sleep?

Reply:
Basically to prevent the user from clicking too fast, which causes the matplotlib doesn't have enough time to plot the new graph correctly.

Reply:
I understand, and it this case is not critical. But that is not the way to prevent these interactions. You should put the reestriction either in the plot or put/start a timer in this method, so that 1s later is re-enabled the button

Reply:
Commit 82fa3b9 fixs the issue.

Reply:
I will be accepting this one for now, but please let's avoid these magic sleeps whenever possible.
There are plenty solutions for these problems

Reply:
@vividf